src/main/webapp/file.xhtml

```diff
@@ -109,6 +109,7 @@
                                         </button>
                                         <ul class="dropdown-menu pull-right text-left" role="menu">
                                             <ui:fragment rendered="#{systemConfig.provCollectionEnabled}">
+                                            <!--  Should be FilePage.isLockedFromEdits() ? -->
```

Comment:
Good question. Is this a question? I will look into this more.

Reply:
~yes. Seems like a cut/paste error but separate from what I was working on.